P5/Source/Specs/att.cmc.xml

```diff
@@ -0,0 +1,145 @@
+      
+      <exemplum type="Chat2CLARIN" xml:lang="en" versionDate="2021-09-07">
+        <p>automatic system message in chat: user moves on to another
+        chatroom</p>
+        <egXML xmlns="http://www.tei-c.org/ns/Examples">
+          <post type="event" generatedBy="system" who="#system" rend="color:blue">
+            <p>
+              <name type="nickname" corresp="#A02">McMike</name> geht
+              in einen anderen Raum: <name type="roomname">Kreuzfahrt</name>
+            </p>
+          </post>
+        </egXML>
+      </exemplum>
+      
+      <exemplum type="Chat2CLARIN" xml:lang="en" versionDate="2021-09-07">
+        <p>automatic system message in chat: user enters a chatroom</p>
+        <egXML xmlns="http://www.tei-c.org/ns/Examples">
+          <post type="event" generatedBy="system">
+            <p>
+              <name type="nickname" corresp="#A08">c_bo</name> betritt
+            den Raum. </p>
+          </post>
+        </egXML>
+      </exemplum>
+      
+      <exemplum type="Chat2CLARIN" xml:lang="en" versionDate="2021-09-07">
+        <p>automatic system message in chat: user changes his font color</p>
+        <egXML xmlns="http://www.tei-c.org/ns/Examples">
+          <post type="event" generatedBy="system" rend="color:red">
+            <p>
+              <name type="nickname" corresp="#A08">c_bo</name> hat die
+              Farbe gewechselt.
+            </p>
+          </post>
+        </egXML>
+      </exemplum>
+      
+      <exemplum type="wikitalk" xml:lang="en" versionDate="2021-09-07">
+        <p>An automatic signature of user including an automatic timestamp
+        (Wikipedia discussion, anonymized). The specification of
+        <att>generatedBy</att> at the inner element <gi>signed</gi> is
+        meant to override the specification at the outer element
+        <gi>post</gi>. This is generally possible when the outer
+        <att>generatedBy</att> value is "human".</p>
+        <egXML xmlns="http://www.tei-c.org/ns/Examples">
+          <post type="standard" generatedBy="human" indentLevel="2"
+                synch="#t00394407" who="#WU00005582">
+            <p> Kurze Nachfrage: Die Hieros für den Goldnamen stammen
+            auch von Beckerath gem. Literatur ? Grüße --</p><signed
+            generatedBy="template" rend="inline"><gap reason="signatureContent"
+            /><time generatedBy="template">18:50, 22. Okt. 2008 (CEST)</time></signed>
+          </post>
+        </egXML>
+      </exemplum>
+      
+      <exemplum type="usenet" xml:lang="en" versionDate="2021-09-07">
+        <p>Usenet news message: a client-generated line that introduces a
+        quotation from a previous message (similar to email):</p>
+        <egXML xmlns="http://www.tei-c.org/ns/Examples">
+          <cit type="replyCit">
+            <bibl type="introQuote" generatedBy="template">Am 03.04.2015 um
+            09:46 schrieb [_NAME_]:</bibl>
+            <quote>
+              <!-- [...] -->
```

Comment:
please change to: [...] without the XML comment

Reply:
fixed at f0e7d1e26.